app/controllers/example_controller.rb

```diff
@@ -12,6 +12,18 @@ def example3
   end
 
   def example4
```

Comment:
Feel free to grab just fizzbuzz out of here and move it into either of the other controllers